examples/using-redux-w-interaction-code-splitting/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "dependencies": {
+    "gatsby": "^3.0.0",
+    "gatsby-image": "^3.0.0",
```

Comment:
Here are some unused packages in the package.json. image, offline, source-filesystem are all not used